internal/daemon/controller/handlers/credentiallibraries/credentiallibrary_service_test.go

```diff
@@ -101,12 +121,6 @@ func TestList(t *testing.T) {
 			res:     &pbs.ListCredentialLibrariesResponse{Items: wantLibraries[1:2]},
 			anonRes: &pbs.ListCredentialLibrariesResponse{Items: wantLibraries[1:2]},
 		},
-		{
-			name:    "Filter on Attribute",
-			req:     &pbs.ListCredentialLibrariesRequest{CredentialStoreId: store.GetPublicId(), Filter: fmt.Sprintf(`"/item/attributes/path"==%q`, wantLibraries[2].GetVaultCredentialLibraryAttributes().GetPath().Value)},
-			res:     &pbs.ListCredentialLibrariesResponse{Items: wantLibraries[2:3]},
-			anonRes: &pbs.ListCredentialLibrariesResponse{}, // anonymous user does not have access to attributes
-		},
```

Comment:
Why was this removed?

Reply:
I remember now. I moved the filter on attributes tests to [TestList_Attributes](https://github.com/hashicorp/boundary/blob/45a51ee23c33cb379065569f26a5ae8c693b007a/internal/daemon/controller/handlers/credentiallibraries/credentiallibrary_service_test.go#L1773) because something was messing up the filter order when I put both library types in the same list/store. TestList_Attributes is exactly the same as TestList, but the different CL types are in separate credential stores.